src/Module.php

```diff
@@ -412,4 +432,18 @@ protected function defaultVersion()
         }
         return parent::defaultVersion();
     }
+
+    /**
+     * @return string
+     */
+    public function htmlTitle(){
+        if(is_string($this->pageTitle) && !empty($this->pageTitle))
+            return $this->pageTitle;
+        $url = Url::base(true);
+        if(is_array($this->pageTitle) && array_key_exists($url, $this->pageTitle))
+            return $this->pageTitle[$url];
+        elseif (is_callable($this->pageTitle))
+            return call_user_func($this->pageTitle, $url);
+        return 'Yii Debugger';
+    }
```

Comment:
PSR-2 formatting would be:

```php
    public function htmlTitle()
    {
        if (is_string($this->pageTitle) && !empty($this->pageTitle)) {
            return $this->pageTitle;
        }

        $url = Url::base(true);

        if (is_array($this->pageTitle) && array_key_exists($url, $this->pageTitle)) {
            return $this->pageTitle[$url];
        } else if (is_callable($this->pageTitle)) {
            return call_user_func($this->pageTitle, $url);
        }

        return 'Yii Debugger';
    }
```

Reply:
I thought using array could be more easier if we don't have much urls,
but now I see that both will be the same